src/Sulu/Bundle/AdminBundle/Resources/js/components/Snackbar/Snackbar.js

```diff
@@ -17,11 +20,38 @@ const ICONS = {
     warning: 'su-bell',
 };
 
-export default class Snackbar extends React.Component<Props> {
+@observer
+class Snackbar extends React.Component<Props> {
     static defaultProps = {
         visible: true,
     };
 
+    @observable message: ?string;
```

Comment:
what is the reason for storing the message as class property instead of just using the prop value? 🤔

Reply:
Because if we don't do that, then the message will disappear before the snackbar is animated away.